### ISSUE TITLE
GAN Discriminator loss wildly variant - expected?

### ISSUE THREAD

**Issue by OptimizerPrime** [deepfakes/faceswap-model#14]
I was trying the GAN model, and the loss values for the discriminator (ie, Loss_DA and Loss_DB) are fluctuating a lot.  Every iteration, it jumps randomly between a value somewhere in the range from < .01 to .25.  I know it's essentially a binary decision, so more random variation from run to run might be expected, but this seems somewhat extreme, and makes me worry that I'm not getting useful derivatives.

Is this normal?  Should I adjust some hyperparameters? (Like batch size, for instance? It's currently 64...)

I should note that I've only been running the training for a few hours thus far, so it's possible things are working as they should - the variation just seemed concerning.  The generator seems to be more stable, and has loss values which are generally trending downward, which is good.

(BTW, not sure if this is the right place to post general "discussion" stuff like this - please let me know if there's some other forum / reddit / whatever I should go for similar questions!)

**Comment by Clorr** [comment]
Hi, for questions about using the GAN version, you should ask directly in the https://github.com/shaoanlu/faceswap-GAN repo

**Comment by OptimizerPrime** [comment]
The issue is directly related to your folk's port of it, not the original code.  I have since tried using his work directly, and it works great.
